build.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+    <!--
+     This project is using the Contao Community Alliance Build System (CCABS)
+
+     For documentation visit:
+       https://github.com/contao-community-alliance/build-system
+
+    -->
+<project name="metamodels-core" default="build">
```

Comment:
You should check that `name` attribute – probably in a couple of other repositories, too. Was set to `metamodels-core` in contao-composer as well. I know it's not big issue at all, though should be fixed.